website/content/docs/enterprise/pkcs11-provider/index.mdx

```diff
@@ -9,6 +9,9 @@ description: |-
 
 # PKCS#11 Provider
 
+-> **Note**: This secret engine requires [Vault Enterprise](https://www.hashicorp.com/products/vault/)
```

Comment:
PKCS#11 provider isn't a secret engine though, right? 

Maybe: this functionality depends on the KMIP secrets engine, which is part of...